### DISCUSSION TITLE
Very newbie question here....

### DISCUSSION THREAD

**Discussion by robbhar** [NginxProxyManager/nginx-proxy-manager#3214] [Q&A]
System - Casaos on Zimaboard
Installed nginx in a docker container, when rebooted can no longer get to Casaos as it keeps showing me the "Congratulations" screen saying that the "host isn't set up yet".  It further says I should login to the admin panel to get started.  I have tried the IP address with ":81/login" and "127.0.0.1:81/login" but all I get is "can't establish a connection".  Is there another way to get to the admin panel or is there a way to get rid of nginx so I can get my CasaOS screen back?
I have searched but not found an answer to this.  Be gentle I am a newbie to docker stuff.
Oh, BTW the other docker containers I have set up on the casaos side are all still accesible.
Thanks,
Rob

**Comment by lastsamurai26** [comment]
can you give us more informations ? 

CasaOS is a Website or what Exectly ?
can you check that the Docker of nginx Proxymanager is startet ? -> docker ps if not try to start the container manually
